config/Sage/game_options.txt

```diff
@@ -564,7 +564,7 @@ RANDOMIZE_SHIFT_TIME
 #SHIFT_TIME_REALTIME
 
 ## A cap on how many monkeys may be created via monkey cubes
-MONKEYCAP 64
+MONKEYCAP 150
```

Comment:
```suggestion
MONKEYCAP 100
```

Maybe keep Sage down to 100 or so. How often does it even hit the 64 cap we currently have?

Reply:
I've never seen anyone meet the cap.